conf/modules/annotate_structural_variants.config

```diff
@@ -37,14 +37,14 @@ process {
 
     withName: '.*ANNOTATE_STRUCTURAL_VARIANTS:ENSEMBLVEP_SV' {
         ext.args = { [
-            '--dir_cache vep_cache',
-            '--dir_plugins vep_cache/Plugins',
+            '--dir_cache cache',
+            '--dir_plugins cache/Plugins',
             '--plugin pLI,pLI_values.txt',
             '--appris --biotype --buffer_size 100 --canonical --cache --ccds',
             '--compress_output bgzip --distance 5000 --domains',
             '--exclude_predicted --force_overwrite --format vcf',
-            '--fork 4 --hgvs --humdiv --max_sv_size 248956422 --merged',
```

Comment:
no forks?

Reply:
Yeah, the script automatically substitutes the number of cpus allocated to the process to `--fork`. This one was redundant.

Reply:
Nice!